### PR TITLE
Update financial-statement-follow-up.vbs #564

### PR DESCRIPTION
**Pull request by KallistaImdieke**
Made corrections so that the CAAD note updates for both parties when you select both.

### CODE REVIEW

**Comment by theVKC** [comment]
@KallistaImdieke: not totally sure what issue this is referring to. Can you make sure your pull request title contains the issue number?

If it doesn't have one, please create one prior to making any pulls (we need to take the time to review the enhancement request or bug status before prioritizing).

**Comment by KallistaImdieke** [comment]
this is in regards to #564 it resolves the error where it didn't update the second CAAD note

**Comment by KallistaImdieke** [comment]
did you need me to do any corrections on this one?

File: actions/financial-statement-follow-up.vbs [hunk 1]
```diff
@@ -230,22 +230,25 @@ IF recipient_code = "BOTH - CP and NCP" THEN
...
-	EMSetCursor 16, 004
-	Call Write_variable_in_CAAD("Finacial Statement follow up letter send to CP")
-	transmit
-	F3
-	EMWriteScreen "M", 3, 029
-	EMSetCursor 16, 004
-	Call Write_variable_in_CAAD("Finacial Statement follow up letter send to NCP")
-	Transmit
-	F3
+CALL navigate_to_PRISM_screen("CAAD")							'modifying and noting the caad note that the letter was sent
```
Comment (theVKC) [inline actions/financial-statement-follow-up.vbs]: @KallistaImdieke, can these be indented please? Anything before the end if should be indented.

Comment (theVKC) [inline actions/financial-statement-follow-up.vbs]: (233-247)

**Pull request closed without merging**